test/data/http/update-rhel-idm-domain.json

```diff
@@ -2,20 +2,20 @@
   "domain_name": "mydomain.example",
   "domain_type": "rhel-idm",
   "rhel-idm": {
-    "realm_name": "MYDOMAIN.EXAMPLE",
+    "realm_name": "UPDATE-MYDOMAIN.EXAMPLE",
```

Comment:
It is not possible to modify the Kerberos realm or domain name of an IdM deployment. We should treat a change as an error. There are probably some other fields that we should treat as read-only after initial registration. Can you open a ticket for the problem?

Reply:
thanks mate! I have created a ticket for it!